### PR TITLE
Job Limit Increases

### PR DESCRIPTION
**Pull request by BoundCNapalm**
Increased job limits for all reasonable jobs.
Attempted to make these changes consistent with Pull Request #29 Makes spawn positions more consistent. Every reasonable position in my eyes has gotten a job number increase. Numbers can be adjusted to taste (More jobs for any one department possible). Detective change is postponed due to uncertainly relating to revolver map spawns. These should be ready for immediate implementation, please note if they are not and where.

### CODE REVIEW

File: jobconfig.toml [hunk 1]
```diff
@@ -16,8 +16,10 @@
 ## "# Total Positions" should always be changed to "Total Positions", no additional spacing.
 ## Best of luck editing!
 
+## Edit as of fork "Job-Limit-Increases": All jobs that I considered reasonable to have a limit increase have been increased.
+
 [AI]
-"Playtime Requirements" = 2400
+"Playtime Requirements" = 1200
```
Comment (Cyprex) [inline jobconfig.toml:22]: Don't include changes from other independent PRs like this- instead let git handle merging the two together.

File: jobconfig.toml [hunk 2]
```diff
@@ -109,49 +111,49 @@
...
+"Spawn Positions" = 2
+"Total Positions" = 2
 
 [CURATOR]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 1
-"Total Positions" = 1
+"Spawn Positions" = 2
+"Total Positions" = 2
```
Comment (Cyprex) [inline jobconfig.toml:127]: Icebox/Northstar have 1, meta/biodome have two. I'm unsure about this one, personally I feel two are weird, but I honestly don't know. There's also the unique whip that I think only one can get.

File: jobconfig.toml [hunk 3]
```diff
@@ -109,49 +111,49 @@
...
 
 [CUSTOMS_AGENT]
 "Playtime Requirements" = 360
 "Required Account Age" = 0
-"Spawn Positions" = 1
-"Total Positions" = 1
+"Spawn Positions" = 2
+"Total Positions" = 2
```
Comment (Cyprex) [inline jobconfig.toml:133]: This is dept. sec- keep this as-is. Also .
This goes for the other dept. sec slots as well, which I won't comment on individually

File: jobconfig.toml [hunk 4]
```diff
@@ -31,8 +33,8 @@
 [ATMOSPHERIC_TECHNICIAN]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 1
-"Total Positions" = 2
+"Spawn Positions" = 3
+"Total Positions" = 3
```
Comment (Cyprex) [inline jobconfig.toml:37]: This here is fine- but in general our maps might not support multiple people spawning with the job, as well as not having the equipment etc.
I will mark occurences of this with just `.` from here on

File: jobconfig.toml [hunk 5]
```diff
@@ -85,7 +87,7 @@
 [CHEMIST]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 1
+"Spawn Positions" = 2
```
Comment (Cyprex) [inline jobconfig.toml:90]: Probably fine, not sure if maps have this- I'd keep it for now.

Comment (Cyprex) [reply jobconfig.toml:90]: Fine on icebox

File: jobconfig.toml [hunk 6]
```diff
@@ -109,49 +111,49 @@
...
 "Total Positions" = 2
 
 [CORRECTIONS_OFFICER]
 "Playtime Requirements" = 450
 "Required Account Age" = 7
-"Spawn Positions" = 1
-"Total Positions" = 1
+"Spawn Positions" = 2
+"Total Positions" = 2
```
Comment (Cyprex) [inline jobconfig.toml:121]: .
I can't tell how many icebox has easily because I believe they use the automapper

File: jobconfig.toml [hunk 7]
```diff
@@ -109,49 +111,49 @@
 [COOK]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 1
+"Spawn Positions" = 2
```
Comment (Cyprex) [inline jobconfig.toml:114]: Should be fine, icebox has 3 spawning positions

File: jobconfig.toml [hunk 8]
```diff
@@ -73,8 +75,8 @@
 [CARGO_TECHNICIAN]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 2
-"Total Positions" = 2
+"Spawn Positions" = 5
```
Comment (Cyprex) [inline jobconfig.toml:78]: Icebox actually only has 3, not sure how to feel about this change. 5 total job slots are fine.

Comment (Cyprex) [reply jobconfig.toml:78]: icebox/biodome 3, northstar 4, meta 5

File: jobconfig.toml [hunk 9]
```diff
@@ -109,49 +111,49 @@
...
+"Spawn Positions" = 2
+"Total Positions" = 2
 
 [CYBORG]
 "Playtime Requirements" = 120
 "Required Account Age" = 0
-"Spawn Positions" = 3
-"Total Positions" = 3
+"Spawn Positions" = 5
+"Total Positions" = 5
```
Comment (Cyprex) [inline jobconfig.toml:139]: icebox/biodome 1, northstar 3, meta 4. Honestly just leave this as-is, let's see what happens.

File: jobconfig.toml [hunk 10]
```diff
@@ -109,49 +111,49 @@
...
 
 [DETECTIVE]
 "Playtime Requirements" = 300
 "Required Account Age" = 0
-"Spawn Positions" = 1
-"Total Positions" = 1
+"Spawn Positions" = 2
+"Total Positions" = 2
```
Comment (Cyprex) [inline jobconfig.toml:145]: Only one locker with gear, maps only really support one detective.

File: jobconfig.toml [hunk 11]
```diff
@@ -169,20 +171,20 @@
...
+"Spawn Positions" = 2
 "Total Positions" = 2
 
 [MEDICAL_DOCTOR]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 3
-"Total Positions" = 3
+"Spawn Positions" = 6
+"Total Positions" = 6
```
Comment (Cyprex) [inline jobconfig.toml:187]: Tons of spawner slots for these, but I think 5 is more appropriate

File: jobconfig.toml [hunk 12]
```diff
@@ -169,20 +171,20 @@
 [JANITOR]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 1
+"Spawn Positions" = 2
```
Comment (Cyprex) [inline jobconfig.toml:174]: northstar/delta support 4, meta/biodome/icebox only 1. Let's just do this, I guess.

File: jobconfig.toml [hunk 13]
```diff
@@ -235,20 +237,20 @@
 [ROBOTICIST]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 1
-"Total Positions" = 2
+"Spawn Positions" = 3
+"Total Positions" = 3
```
Comment (Cyprex) [inline jobconfig.toml:241]: Keep this at two, I feel like that's already a bit much.

File: jobconfig.toml [hunk 14]
```diff
@@ -259,8 +261,8 @@
 [SECURITY_OFFICER]
 "Playtime Requirements" = 300
 "Required Account Age" = 0
-"Spawn Positions" = 4
-"Total Positions" = 4
+"Spawn Positions" = 7
+"Total Positions" = 7
```
Comment (Cyprex) [inline jobconfig.toml:265]: Keep this as-is, this is highly sensitive to balance and should be discussed in a separate PR.

File: jobconfig.toml [hunk 15]
```diff
@@ -271,8 +273,8 @@
 [STATION_ENGINEER]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 2
-"Total Positions" = 3
+"Spawn Positions" = 6
+"Total Positions" = 6
```
Comment (Cyprex) [inline jobconfig.toml:277]: 4-5 is enough imo. 5 is also the lowest our maps go in terms of starting slots

File: jobconfig.toml [hunk 16]
```diff
@@ -283,8 +285,8 @@
 [VIROLOGIST]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 1
-"Total Positions" = 1
+"Spawn Positions" = 2
+"Total Positions" = 2
```
Comment (Cyprex) [inline jobconfig.toml:289]: There's only enough machinery around for one virologist to comfortably work- keep this as-is.

**Review by Cyprex — CHANGES_REQUESTED** [review]

File: jobconfig.toml (hunk 1 above)
Comment (BoundCNapalm) [reply jobconfig.toml:22]: Next commit reverts this change.

File: jobconfig.toml (hunk 2 above)
Comment (BoundCNapalm) [reply jobconfig.toml:127]: Next commit reduces curator total to 1.

File: jobconfig.toml (hunk 3 above)
Comment (BoundCNapalm) [reply jobconfig.toml:133]: Acknowledged; Next commit reduces all active dept guard totals to 1.

File: jobconfig.toml (hunk 4 above)
Comment (BoundCNapalm) [reply jobconfig.toml:37]: Reduced spawn positions to 1 in next commit just for sake of game stability, incase such causes issues. When the next commit gets ingame testing, may adjust value in a new PR to better follow 29.

File: jobconfig.toml (hunk 6 above)
Comment (BoundCNapalm) [reply jobconfig.toml:121]: Looked at this and reduced spawn positions from 2 to 1 in next commit. If confirmed that 2 is a value okay with maps like Icebox, then will raise it again in a new PR.

File: jobconfig.toml (hunk 8 above)
Comment (BoundCNapalm) [reply jobconfig.toml:78]: Reducing cargo technician spawn positions to 3 to prevent problems. If testing with another commit proves that this spawn position increase is okay, then will raise it back to the intended 5.

File: jobconfig.toml (hunk 10 above)
Comment (BoundCNapalm) [reply jobconfig.toml:145]: Reverted back to 1. New PR to increase spawn changes following balance discussion will be taken into consideration in potential follow-up PR after testing.

File: jobconfig.toml (hunk 11 above)
Comment (BoundCNapalm) [reply jobconfig.toml:187]: Reduced total slot amount to 5 in next commit.

File: jobconfig.toml (hunk 12 above)
Comment (BoundCNapalm) [reply jobconfig.toml:174]: Reduced spawn position amount to 1 for janitor in next commit; But will see what happens.

File: jobconfig.toml (hunk 13 above)
Comment (BoundCNapalm) [reply jobconfig.toml:241]: Understood; Reduced to 2.

File: jobconfig.toml (hunk 14 above)
Comment (BoundCNapalm) [reply jobconfig.toml:265]: Change reverted, will change in accordance to balance discussion alongside Detective.

File: jobconfig.toml (hunk 15 above)
Comment (BoundCNapalm) [reply jobconfig.toml:277]: Altered to 5 in both counts; Will see how this changes balance.

File: jobconfig.toml (hunk 16 above)
Comment (BoundCNapalm) [reply jobconfig.toml:289]: Change reverted in next commit.

File: README.md [hunk 17]
```diff
@@ -1,9 +1,7 @@
-# Bubberstation config overrides
-On the live server these files replace any files of the same name within the config/ subfolder of the repo. 
-This is necessary because testing on localhost necessarily requires different configs than the live servers.
-To facilitate this this repo is pulled into GameStaticFiles/config/ by TGS at the end of every round(via a helper EventScript).
-Actually this is not the case yet, due to difficulties in doing so...
+# Bubberstation Job Limits Increase Fork
+This is a fork to increase job limits as indicated by the name.
```
Comment (Cyprex) [inline README.md:2]: Any changes made yon your branch will be in the PR- to avoid this you're supposed to use branches but here I'd just edit it back. The idea is that you make a branch(which is functionally a separate version of the files) for each PR you make and then PR from that branch.

File: jobconfig.toml [hunk 18]
```diff
@@ -16,6 +16,8 @@
 ## "# Total Positions" should always be changed to "Total Positions", no additional spacing.
 ## Best of luck editing!
 
+## Edit as of fork "Job-Limit-Increases": All jobs that I considered reasonable to have a limit increase have been increased.
```
Comment (Cyprex) [inline jobconfig.toml:19]:
```suggestion
```

File: jobconfig.toml (hunk 4 above)
Comment (Cyprex) [reply jobconfig.toml:37]: 2 starting should be fine, 3 too, probably.

File: jobconfig.toml (hunk 6 above)
Comment (Cyprex) [reply jobconfig.toml:121]: This wasn't actually reduced to 1

File: README.md (hunk 17 above)
Comment (BoundCNapalm) [reply README.md:2]: Apologies, literally first PR. I'll fix that by reverting the file
